### PR TITLE
[2.3] SMTP

### PR DESCRIPTION
**Pull request by chipzoller**
Signed-off-by: chipzoller <chipzoller@gmail.com>

## Proposed Changes

Adds docs on the custom SMTP configuration feature available in 2.3.

### CODE REVIEW

File: install-and-configure/advanced-configuration/smtp.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Custom SMTP Configuration
+
+Kubecost Enterprise supports defining a custom SMTP server which, when defined, will be preferred by all Kubecost scheduled reports and alerts when sending email. Use of a custom SMTP server is especially beneficial in air gapped or hardened computing environments when the default, public SMTP server used by Kubecost cannot be contacted.
```
[the remark this answers is not in the document]
Comment (chipzoller) [reply install-and-configure/advanced-configuration/smtp.md:3]: The option won't even appear if the license isn't enterprise.

[the remark this answers is not in the document]
Comment (kwombach12) [reply install-and-configure/advanced-configuration/smtp.md:3]: very cool!

**Review by kwombach12 — APPROVED** [review]
LGTM!

**Comment by chipzoller** [comment]
Updating base branch to `v2.3-content` since we've already got 2.3 content there. Merging.

**Pull request merged**